.github/workflows/ci.yml

```diff
@@ -4,14 +4,14 @@ on: [push, pull_request]
 
 jobs:
   run:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
What is `ubuntu-latest`? Shouldn't it be version 20?

Reply:
This is a personal preference. I force using LTS only and be sure no problem is due to docker host

Reply:
I see. Let's stick with latest for now.